umm-spec-lib/resources/json-schemas/variable/umm/v1.9/umm-var-json-schema.json

```diff
@@ -0,0 +1,604 @@
+          "type": "string",
+          "minLength": 0,
+          "maxLength": 4000
+        }
+      },
+      "required": ["SamplingMethod", "MeasurementConditions"]
+    },
+    "FillValueType": {
+      "description": "The fill value, fill value type and fill value description of the variable in the data file. The fill value is generally a value which falls outside the valid range. For example, if the valid range is '0, 360', the fill value may be '-1'. The elements of this section apply to the fill value of a variable.",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "Value": {
+          "description": "The fill value of the variable in the data file.",
+          "type": "number"
+        },
+        "Type": {
+          "description": "Type of the fill value of the variable in the data file.",
+          "type": "string",
+          "enum": ["SCIENCE_FILLVALUE", "QUALITY_FILLVALUE", "ANCILLARY_FILLVALUE", "OTHER"]
+        },
+        "Description": {
+          "description": "Description of the fill value of the variable in the data file.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 160
+        }
+      },
+      "required": ["Value", "Type"]
+    },
+    "SetType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "The elements of this section apply to variable sets.",
+      "properties": {
+        "Name": {
+          "description": "This element enables specification of set name. For example, 'Data_Fields'.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 80
+        },
+        "Type": {
+          "description": "This element enables specification of set type. For example, if the variables have been grouped together based on a particular theme, such as wavelength, then the type should be set to that theme, otherwise it should be set to 'General'.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 80
+        },
+        "Size": {
+          "description": "This element specifies the number of variables in the set. For example, if the number of variables in the set is fifteen, the size should be set to '15'.",
+          "type": "number"
+        },
+        "Index": {
+          "description": "This element specifies the index value within the set for this variable, For example, if this varible is the third variable in the set, the index value should be set to '3'.",
+          "type": "number"
+        }
+      },
+      "required": ["Name", "Type", "Size", "Index"]
+    },
+    "ValidRangeType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "Valid range data value of a variable: minimum and maximum values. For example, '-100, 5000'.",
+      "properties": {
+        "Min": {
```

Comment:
I assume there is no real limit to what people enter in?

Reply:
No, the variable limits depend on the science teams that created the sensor and the sensor limits.